src/freenet/store/saltedhash/SaltedHashFreenetStore.java

```diff
@@ -1070,10 +1070,11 @@ private void setStoreFileSize(long storeMaxEntries) {
 				{
 					wrapperKeepalive.start();
 					if (oldMetaLen < newMetaLen) {
-						Fallocate.forChannel(metaFC, newMetaLen).fromOffset(oldMetaLen).execute();
+						// freenet-mobile-changed: Passing file descriptor to avoid using reflection
+						Fallocate.forChannel(metaFC, metaRAF.getFD() ,newMetaLen).fromOffset(oldMetaLen).execute();
```

Comment:
this is purely stylistic: here there’s a space before the , instead of after it.

Reply:
I did that change myself now.